std/file.d

```diff
@@ -2865,7 +2865,7 @@ version(Windows) string getcwd()
         return ptr[0 .. n2].to!string;
     }
 }
-else version (Solaris) string getcwd()
+else version (Solaris) string getcwd() @trusted
```

Comment:
In this branch and for Posix, the function is simple enough where this can be applied without much worry.